MoviesStoreApp/MoviesStoreMVVM/ViewModel/MovieInfoViewModelProtocol.swift

```diff
@@ -0,0 +1,19 @@
+// MovieInfoViewModelProtocol.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Протокол вью модели экрана с информацией о фильме
+protocol MovieInfoViewModelProtocol {
+    // MARK: - Public Properties
+
+    var updateView: VoidHandler? { get set }
+    var showErrorAlert: ErrorHandler? { get set }
+    var movieID: Int? { get set }
+    var movieInfo: MovieInfo? { get set }
```

Comment:
```suggestion
    var movieID: Int? { get set }
    var movieInfo: MovieInfo? { get set }
```
  все ли свойства торчащие наружу должны быть изменяемыми?